README.md

```diff
@@ -55,6 +55,8 @@ resource_types:
       -----END RSA PRIVATE KEY-----
     ```
 
+* `assignee`: *Optional.* If specified, add this username to the assignee list.
```

Comment:
Please change to `assignee_file` and have docs reflect that this is taking in a path to a file, not the assignee name itself.

Reply:
Done.